README.md

```diff
@@ -1,3 +1,17 @@
-# Skynet SDK
+# Skynet NodeJS SDK
```

Comment:
We should add some examples to this document and expose it on GitHub pages like https://nebulouslabs.github.io/skynet-js/ (it is a setting in the repository options)

Reply:
Created https://github.com/NebulousLabs/nodejs-skynet/issues/30